Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM registry-proxy.engineering.redhat.com/rh-osbs/rhacm2-yarn-builder:v2.3.0_14-1.20210810163126 AS builder
+ARG TARGET_BRANCH=master
+
+RUN dnf install -y git
+RUN git clone https://github.com/SanjalKatiyar/dfr-console.git
+WORKDIR /dfr-console
```

Comment:
Do you need to upstream here ?
git remote add upstream GIT_REPO ?

Reply:
I am not sure I understood correctly (sorry for that)...
this is initial Dockerfile, right now there is only one repository under my personal Github account we haven't migrated to `red-hat-storage` yet... I will update this path once we have migrated...